docs: Added steps for Firebase Service ID field in INITIAL_SETUP (#359)
invertase/react-native-apple-authentication: docs/INITIAL_SETUP.md

![complete key registration](images/complete-registration.png)

### Firebase Setup

> If using `@react-native-firebase/auth` for Social Login with Apple, for apple devices
> To make `auth().signInWithCredential(appleCredential)` to work, add app's bundle id in the `Service ID` field.

Reviewer: ```suggestion
If you will use `@react-native-firebase/auth` for Social Login with Apple for Apple devices, you must add the Apple app's bundle id in the `Service ID` field in the Firebase Console
```
